corehq/apps/hqcase/api/get_list.py

```diff
@@ -116,10 +116,12 @@ def get_list(domain, couch_user, params):
     }
 
     cases_in_result = len(hits)
-    if cases_in_result and es_result.total > cases_in_result:
+    limit = query._size or MAX_PAGE_SIZE
+    if cases_in_result == limit:
```

Comment:
I don't follow - won't this still give us a blank page at the end if the number of results is a multiple of page size?

Reply:
Not saying that's a problem, just that the test changes an the commit message made it sound like that would no longer be the case.  Also, why can't this look at the total number of results any more?

Reply:
commented too early - the next commit adds a test that spells it out clearly, though I'm still curious why looking at `result.total` is no longer viable.

Reply:
My understanding is that previously we used range queries which filtered down the total results, but `search_after` instead returns the total number of documents that match the underlying query, and `search_after` just says where to start from. So if there are 11 cases and your page size is 5, the `es_result.total` is going to be 11 every time instead of 5, 5, 1.